Package.swift

```diff
@@ -4,15 +4,14 @@ import PackageDescription
 let package = Package(
     name: "swift-otel-semantic-conventions",
     products: [
-        .library(name: "OTelSemanticConventions", targets: ["OTelHTTPTracingConventions"]),
         .library(name: "OTelSemanticMetricsConventions", targets: ["OTelHTTPMetricsConventions"]),
-        .library(name: "OTelSemanticTracingConventions", targets: ["OTelHTTPTracingConventions"]),
         .library(
             name: "OTelHTTPConventions",
-            targets: ["OTelHTTPMetricsConventions", "OTelHTTPTracingConventions"]
+            targets: ["OTelHTTPMetricsConventions"]
         ),
         .library(name: "OTelHTTPMetricsConventions", targets: ["OTelHTTPMetricsConventions"]),
```

Comment:
Do we need this granular targets/products, or can we just put everything into a single target and product? If we use traits, I'd be happy with a single module for everything.

Reply:
This is a left-over from before this PR. @NeedleInAJayStack Feel free to delete that target as part of the PR too.

Reply:
Sure, I've removed the Metrics libraries here: https://github.com/swift-otel/swift-otel-semantic-conventions/pull/2/commits/aa62725410f1dff057599a62428d65d7c16a03dd